hathor/p2p/sync_v2/blockchain_streaming_client.py

```diff
@@ -102,7 +102,6 @@ def handle_blocks(self, blk: Block) -> None:
         if self.tx_storage.partial_vertex_exists(blk.hash):
             # We reached a block we already have. Skip it.
             self._blk_repeated += 1
-            is_duplicated = True
```

Comment:
Why did you remove this? It seems unrelated to the PR and it would change the behavior of sync-v2.

Reply:
This variable is unused in this context. It's just a local variable that we set here, and the method returns just below.

In fact, in the latest commit (https://github.com/HathorNetwork/hathor-core/pull/1165/commits/b9b84ea0802ea3ae98dbc324d8ab07434a2f3575) I completely removed as it's just used to toggle a debug message that would never change.